src/schema/workflow_execution_activity.yaml

```diff
@@ -44,7 +45,47 @@ classes:
       - part_of
       - type
       - version
-
+      - qc_status
+      - qc_comment
+      - has_failure_categorization
+    rules:
```

Comment:
I haven't scrutinized these. if there's example data to exercise all of the conditions that the rules know about, and those examples pass or fail as excepted, then they are OK with me.